libevmcore/Instruction.h

```diff
@@ -127,16 +127,16 @@ enum class Instruction: uint8_t
 	PUSH30,				///< place 30 byte item on stack
 	PUSH31,				///< place 31 byte item on stack
 	PUSH32,				///< place 32 byte item on stack
-
+	
 	DUP1 = 0x80,		///< copies the highest item in the stack to the top of the stack
-	DUP2,				///< copies the second highest item in the stack to the top of the stack
-	DUP3,				///< copies the third highest item in the stack to the top of the stack
-	DUP4,				///< copies the 4th highest item in the stack to the top of the stack
-	DUP5,				///< copies the 5th highest item in the stack to the top of the stack
-	DUP6,				///< copies the 6th highest item in the stack to the top of the stack
-	DUP7,				///< copies the 7th highest item in the stack to the top of the stack
-	DUP8,				///< copies the 8th highest item in the stack to the top of the stack
-	DUP9,				///< copies the 9th highest item in the stack to the top of the stack
+	DUP2,					///< copies the second highest item in the stack to the top of the stack
```

Comment:
`git checkout -p origin/develop Instruction.h` provides you with an interactive tool that lets you revert single chunks.